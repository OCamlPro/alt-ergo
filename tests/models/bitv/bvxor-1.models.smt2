(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 8))
(declare-const y (_ BitVec 8))
(assert (= x (bvxor #b10100101 y)))
(check-sat)
(get-model)
