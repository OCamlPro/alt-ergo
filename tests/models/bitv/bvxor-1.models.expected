
unknown
(
  (define-fun x () (_ BitVec 8) #x00)
  (define-fun y () (_ BitVec 8) #xa5)
)
